PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskTrackSkim.cxx

```diff
@@ -386,7 +393,8 @@ Bool_t AliAnalysisTaskTrackSkim::FillHistograms()
   }
   
   for (auto particle : particles->accepted()) {
-    fFirstTrackCollectionKinematics["pt"].push_back(particle->Pt());
+    if(fChargeScaling && particle->Charge() < 0) fFirstTrackCollectionKinematics["pt"].push_back(particle->Pt()*-1);
+    else fFirstTrackCollectionKinematics["pt"].push_back(particle->Pt()); 
```

Comment:
Please use `{}` even for one line statements - this is only stylistic, but I strong prefer it since it can avoid bugs in the future